### ISSUE TITLE
Unable to convert .heic file

### ISSUE THREAD

**Issue by jnnlcnjs** [bamlab/react-native-image-resizer#369]
**Describe the bug**
Uploading .heic file returns [Error: Unable to load source image from path]

**To Reproduce**
1. Upload any .heic file

**Observed behavior**
Uploading .heic file returns [Error: Unable to load source image from path]

**Expected behavior**
Continue to convert the file like other image files

**Screenshots**
N/A

**Platform concerned:**
 - Device: Emulator Pixel 5
 - OS: Android 13

**Additional context**

**Link to reproducible example**
https://.....

**Comment by GaelCO** [comment]
Hi,
Some users have problems with heic images
I did some tests. The most part of .heic file I found on web works.But, I found an image that is problematic.
The error log : Error: Unable to load source image from path

For information, I did my test on android and the android viewer can't load the file too. But Gimp on my PC can.
The next week I try to test on a real iPhone and give an iPhone .heic.

I Zip the file, because heic format was not authorized by github : 

[heic-file-sample.zip](https://github.com/bamlab/react-native-image-resizer/files/11848501/heic-file-sample.zip)

**Comment by GaelCO** [comment]
My issue reported by ios users did not come from the image itself.
It came from the path that didn't start with file://. This path is returned by another library.
I workaround it by adding this prefix on iOS

**Comment by taboulot** [comment]
@jnnlcnjs,

A lot of path/uri problem has been fixed in `3.0.6`.
Did a test [here](https://github.com/taboulot/TestImageResizer/tree/rn-image-resizer-3.0.6-heic) and everything works fine.

Here is how you can upload a HEIC image on your emulator:
```sh
./push-heic-android.sh
# Then open the app and you should be able to resize the uploaded image
```

Until you provide me a fully reproducible example, i close this issue for now.

**Comment by GaelCO** [comment]
Hi, the PR https://github.com/bamlab/react-native-image-resizer/pull/381 has fixed my uri issue on iOS.
I have removed my workaround after update react-native-image-resizer to version 3.0.7.
Great job. Thanks
